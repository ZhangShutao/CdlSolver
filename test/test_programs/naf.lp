a :- &c{a}, not b.
b :- &c{b}, not a.