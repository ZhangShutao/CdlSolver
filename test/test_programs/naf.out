default model: 1
x:b; y:b,a;
default model: 2
x:a; y:b,a;
Satisfiable